src/uwtools/drivers/stager.py

```diff
@@ -0,0 +1,56 @@
+"""
+A base class for tasks that stage files.
```

Comment:
```suggestion
A mixin class for tasks that stage files.
```
Just to be clear about our intent.